boltiot/bolt.py

```diff
@@ -158,3 +158,15 @@ def isAlive(self):
         :rtype: JSON
         """
         return request_from(url('isAlive'), self.api_key, self.device_id)
+
+    def isOnline(self):
+        """
+        Check the device status with last updated time
+
+        :param None
+        :returns:  request status, device status: online/offline, time: last of last updated value
+        :example: {"success": "1", "value": "online", "time":"Sun 2018-05-06 08:14:43"}
+
+        :rtype: JSON
+        """
+        return request_from(url('isOnline'), self.api_key, self.device_id)
```

Comment:
@Deepanshu625 Is it better to have a timeout over here? What happens if the server is down from our end?

Reply:
@sashoeb Yeah, but This is an open source library so, I don't think we should do that for one function only. and since this one is for creating a new function for API I think we should keep it similar with others.